core-dev/packages/rocq-stdlib/rocq-stdlib.dev/opam

```diff
@@ -0,0 +1,51 @@
+opam-version: "2.0"
+synopsis: "The Rocq Proof Assistant -- Standard Library"
```

Comment:
Isn't the official name going to be: "The Rocq Prover"?

Reply:
Probably but "Proof Assistant" is what we currently have in each opam file (including rocq-runtime and rocq-core. They should probably be updated all at once in a separate PR.